package.json

```diff
@@ -60,6 +58,9 @@
     "react-native": "^0.63.4"
   },
   "devDependencies": {
-    "react-native-builder-bob": "^0.18.1"
+    "react": "^16.14.0",
+    "react-native": "^0.63.4",
+    "react-native-builder-bob": "^0.18.1",
```

Comment:
This dependency looks big (the `react-native-builder-bob/` directory is pretty deep).  What are the security risks related to this?  Do you know which subpackages are used?  The top-level deps look okay.

Just verifying, will devDependencies only be run on our machines (not end customers?). Or will their dev instances pull in devDependencies?

Reply:
The `react-native-builder-bob` dependency was already there previously. It generates various files (js, tsc, and so on) from our tsx files. These help with things like command completion for our library.

This was part of the template generated by the RN native module config. It has over 700 users on Github. I'm not sure what the security risks are, do you have some specific concerns? It seems to be a popular tool among the community.

The `devDependencies` should only be installed when running `npm i` on a local dev machine. For production builds, `npm i --production` should not install any `devDependencies`.

Reply:
Talked in person. We'll eventually push this step to a Github action and check in the resulting autogenerated files instead of having clients need to do the `bob build` step themselves.